modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,130 @@
+= Audit Logging
+:description: Learn how to use Redpanda's audit logging capabilities.
+
+Many scenarios for streaming data include the need for fine-grained auditing of user activity related to the system. This is especially true for regulated industries such as finance, healthcare, and the public sector. Complying with PCI DSS v4 standards, for example, requires verbose and detailed activity auditing, alerting, and analysis capabilities.
```

Comment:
I suggest linking to the standards that you cite here. https://pcidssguide.com/whats-new-in-pci-dss-v4-0/